docs/setup-and-installing.md

```diff
@@ -29,7 +29,7 @@
 
     #### Windows
 
-	- [Unity 2018.2.0f2](https://unity3d.com/get-unity/download/archive)
+	- [Unity 2018.2.0f2](https://unity3d.com/get-unity/download/archive) with Linux Build Support, Mac Build Support (IL2CPP) and Windows Build Support (Mono) components selected during the installation process.
```

Comment:
Windows doesn't actually have the option to install mac il2cpp, only just "mac" which is mono.
Similarly, Windows build support is not an available module as it is installed by default.